libs/platform/src/lib/components/dynamic-page/constants.ts

```diff
@@ -0,0 +1,54 @@
+    dynamicPageTitleAreaCollapsed: 'fd-dynamic-page__title-area--collapsed', // need to use thiis TODO
+    dynamicPageTitleContainer: 'fd-dynamic-page__title-container',
+    dynamicPageTitle: 'fd-dynamic-page__title',
+    dynamicPageBreadcrumb: 'fd-dynamic-page__breadcrumb',
+    dynamicPageKeyInfo: 'fd-dynamic-page__title-content',
+    dynamicPageGlobalActions: 'fd-dynamic-page__toolbar',
+    dynamicPageLayoutActions: 'fd-dynamic-page__toolbar--actions',
+    dynamicPageHeader: 'fd-dynamic-page__header',
+    dynamicPageCollapsibleHeader: 'fd-dynamic-page__collapsible-header',
+    dynamicPageCollapsibleHeaderTransparentBg: 'fd-dynamic-page__collapsible-header--transparent-bg',
+    dynamicPageCollapsibleHeaderSmall: 'fd-dynamic-page__collapsible-header--sm',
+    dynamicPageCollapsibleHeaderMedium: 'fd-dynamic-page__collapsible-header--md',
+    dynamicPageCollapsibleHeaderLarge: 'fd-dynamic-page__collapsible-header--lg',
+    dynamicPageCollapsibleHeaderExtraLarge: 'fd-dynamic-page__collapsible-header--xl',
+    dynamicPageCollapsibleHeaderPinCollapseNoShadow:
+        'fd-dynamic-page__collapsible-header-visibility-container--no-shadow',
+    dynamicPageTabs: 'fd-dynamic-page__tabs',
+    dynamicPageTabsAddShadow: 'fd-dynamic-page__tabs--add-shadow',
+    dynamicPageTabsSmall: 'fd-dynamic-page__tabs--sm',
+    dynamicPageTabsMedium: 'fd-dynamic-page__tabs--md',
+    dynamicPageTabsLarge: 'fd-dynamic-page__tabs--lg',
+    dynamicPageTabsExtraLarge: 'fd-dynamic-page__tabs--xl',
+    dynamicPageContent: 'fd-dynamic-page__content',
+    dynamicPageContentAreaSmall: 'fd-dynamic-page__content--sm',
+    dynamicPageContentAreaMedium: 'fd-dynamic-page__content--md',
+    dynamicPageContentAreaLarge: 'fd-dynamic-page__content--lg',
+    dynamicPageContentAreaExtraLarge: 'fd-dynamic-page__content--xl',
+    dynamicPageContentListBg: 'fd-dynamic-page__content--list-bg',
+    dynamicPageContentTransparentBg: 'fd-dynamic-page__content--transparent-bg',
+    dynamicPageSummarizedTitleArea: 'fd-dynamic-page__summarized-title-area',
+    dynamicPageSummarizedTitleAreaSmall: 'fd-dynamic-page__summarized-title-area--sm',
+    dynamicPageSummarizedTitleAreaMedium: 'fd-dynamic-page__summarized-title-area--md',
+    dynamicPageSummarizedTitleAreaLarge: 'fd-dynamic-page__summarized-title-area--lg',
+    dynamicPageSummarizedTitleAreaExtraLarge: 'fd-dynamic-page__summarized-title-area',
+    dynamicPageSummarizedTitleAreaNoShadow: 'fd-dynamic-page__summarized-title-area--no-shadow',
+    dynamicPageSummarizedTitle: 'fd-dynamic-page__summarized-title',
+    dynamicPageSummarizedTitleCollapsed: 'fd-dynamic-page__summarized-title-area--collapsed'
+} as const;
+
+export const DYNAMIC_PAGE_CHILD_TOKEN = new InjectionToken<string>('DYNAMIC_PAGE_CHILD_TOKEN');
+
+export type BACKGROUND_TYPE = 'solid' | 'list' | 'transparent';
+
+export type RESPONSIVE_SIZE = 'small' | 'medium' | 'large' | 'extra-large';
```

Comment:
Types should be camel-case, i.e. `BackgroundType` and `ResponsiveSize`.

If these types are unique to Dynamic Page the we should also name space `DynamicPageBackgroundType` and `DynamicPageResponsiveSize`.